mlflow_cratedb/patch/mlflow/model.py

```diff
@@ -9,12 +9,18 @@ def polyfill_uniqueness_constraints():
 
     TODO: Submit patch to `crate-python`, to be enabled by a
           dialect parameter `crate_polyfill_unique` or such.
+
+    TODO: There are two more unique constraints defined on the MLflow data model.
+          - SqlExperimentPermission: "experiment_id", "user_id"
+          - SqlRegisteredModelPermission: "name", "user_id"
     """
     from mlflow.store.model_registry.dbmodels.models import SqlRegisteredModel
     from mlflow.store.tracking.dbmodels.models import SqlExperiment
+    from mlflow.server.auth.db.models import SqlUser
 
     listen(SqlExperiment, "before_insert", check_uniqueness_factory(SqlExperiment, "name"))
     listen(SqlRegisteredModel, "before_insert", check_uniqueness_factory(SqlRegisteredModel, "name"))
+    listen(SqlUser, "before_insert", check_uniqueness_factory(SqlUser, "username"))
```

Comment:
Other than the constraint on `SqlUser` added on behalf of this patch, there are two more to be addressed.

Reply:
The implementation currently is not able to handle composite unique key constraints, so I've created a corresponding issue to track it.

- https://github.com/crate-workbench/cratedb-toolkit/issues/69